[ja] **_ ... _** removed `_`.

juner opened the pull request:
There must be a space to the right of `**_ ... _**`.

### Description

N/A

### Motivation

N/A

### Additional details

N/A

### Related issues and pull requests

N/A

juner committed:
**_ ... _** required space!

mfuji09 commented:
Please tell me why a space must be.

juner commented:
> Please explain why a space must be.

see

![Screenshot_20250113-172619](https://github.com/user-attachments/assets/413ab8b9-e112-4d49-a12f-16aff014e63c)

mfuji09 commented:
@juner 
Typically, Japanese language doesn't put whitespaces between words.
I recommend removing underscores instead of inserting unnecessary whitespace, because Japanese fonts doesn't have the italic variant of fonts.

juner committed:
Response to issues raised

Response to issues raised

> @juner
> Typically, Japanese language doesn't put whitespaces between words.
> I recommend removing underscores instead of inserting unnecessary whitespace, because Japanese fonts doesn't have the italic variant of fonts.
https://github.com/mdn/translated-content/pull/25362#issuecomment-2603592285

juner commented:
@mfuji09 fixed.

mfuji09 reviewed (approved):
OK. Thank you!
